src/mailer/index.ts

```diff
@@ -25,7 +36,7 @@ function sendEmail(body: any): Promise<any> {
 
 function signinEmail(name: string, email: string, token: string, code: string): Promise<any> {
   const body = {
-    from: config.auth.user,
+    from: process.env.GMAIL_USER,
```

Comment:
perhaps we could make the assignment at the top, so we don't have to refer to the `process` object every time and instead have a regular `gmailUser` const